### DISCUSSION TITLE
One device only seeing 1 other node.

### DISCUSSION THREAD

**Discussion by SnickerSnac** [meshtastic/Discussions#90] [General]
I have set up a RP2040 as a stationery node and it can see dosens of other devices. However my T-beam can only see that one node, not any of the other nodes that can see. 
New to this but I was under the impression it should be able to see them all aswell. 
Any suggestions? Thanks

**Comment by fifieldt** [comment]
Are the other nodes your RP2040 is seeing via LoRa or via MQTT?

**Comment by mariotti** [comment]
@SnickerSnac Yes, ideally.

It might be a configuration issue, for example channels settings.

Or device settings.
For example if you set up your RP as REPEATER, and your Beam as CLIENT the view (see) might be pretty different.

Then of course what @fifieldt was mentioning.

What do you mean by see?

A+
F

**Comment by mariotti** [comment]
@SnickerSnac any update? I am interested as going to set up a private mesh.

**Comment by SnickerSnac** [comment]
I had MQTT off and all the setting the same as my Tbeams, client ect. and still loads of problems. I've ordered a helteck and will see if I can get more consistent results with that as it feels like something with the RPs or how I'm setting them up. 
Can update when it arrives

**Comment by mariotti** [comment]
Just in case,

I introduced a new node in my network and it was a little pain to get it visible to the other nodes.
 - at start I did not configured the channels in the same way. :(

So start over.

Then for one specific node I had to reset the credentials, public/private keys.

In practice it took forever to make all my 3 nodes to discover each other.

Then I started to think "window (TM)", when it does not work reboot it.
I had the impression that via serial you can set parameters but do not get into effect
until an actual reboot.

Right now I still have a problem with one node, which I cannot yet physically access,
but the rest seems to work.

I wanted to share a bit of bad experience ;)

Keep us posted
A+
F
